src/main/webapp/dev/js/main/instructorCourseEdit.es6

```diff
@@ -473,6 +473,20 @@ $(document).ready(() => {
         showNewInstructorForm();
     });
 
+    $("[name='instructorisdisplayed']").change((e) => {
+        if ($(e.target).prop('checked')) {
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('readonly', false);
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('value', 'Instructor');
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input')
+                    .attr('placeholder', 'E.g.Co-lecturer, Teaching Assistant');
+        } else {
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('readonly', false);
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input').attr('value', 'Instructor');
+            $(e.target).parents('div.form-group').find('div.col-sm-9 input')
+                    .attr('placeholder', '(This instructor will NOT be displayed to students)');
```

Comment:
This placeholder may not help much given the default value of "Instructor" already being present in the input, preventing this from being seen unless the user manually deletes whatever was in the input.